docs/data-sources/vkcs_images_image.md

```diff
@@ -55,8 +55,6 @@ data "vkcs_images_image" "ubuntu" {
 * `visibility` - (Optional) The visibility of the image. Must be one of 
     "private", "community", or "shared". Defaults to "private".
 
-* `hidden` - (Optional) Whether or not the image is hidden from public list.
```

Comment:
such changes require to be specified in commit title

Reply:
Update commit title

Reply:
Update PR title too, pls